docs/migrations.md

```diff
@@ -0,0 +1,29 @@
+# Schema Migrations
+
+The database schema is versioned and every change requires a migration script to be executed. See [storage/migrations/README.md](storage/migrations/README.md) for more information.
```

Comment:
The migration link 404's for me, but maybe it's a PR/formatting thing?

Reply:
Good catch!